inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/exporter/ExportedKnowledgeBase.java

```diff
@@ -52,6 +54,12 @@
     @JsonProperty("property_type_iri")
     private String propertyTypeIri;
 
+    @JsonProperty("property-label-iri")
```

Comment:
We seem to be mixing naming styles here, i.e. `property_type` vs. `property-label`. One style only please!